### PR TITLE
adding more meta documentation

### PR DESCRIPTION
**Pull request by sbvegan**
**Description**

Adding additional meta documentation to provide additional clarity to the repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,79 +6,104 @@
...
+| [jq](https://github.com/jqlang/jq)                                    | `^1.6`    | `jq --version`        |
+| [just](https://github.com/casey/just?tab=readme-ov-file#installation) | `^1.28.0` | `just --version`      |
 
-### 3. Set env vars
+### 2. Set environment variables
 
-To contribute a standard OP-Stack chain configuration, in addition to user-supplied metadata (chain name) the following data is required: contracts deployment, rollup config, L2 genesis. We provide a tool to scrape this information from your local [monorepo](https://github.com/ethereum-optimism/optimism) folder.
-
-First, make a copy of `.env.example` named `.env`, and alter the variables to appropriate values. Each value is explained in a comment in `.env.example`.
+Make a copy of `.env.example` named `.env`, and alter the variables to appropriate values. Each value is explained in a comment in `.env.example`.
```
Comment (tessr) [inline README.md:69]: ❤️

File: README.md [hunk 2]
```diff
@@ -6,79 +6,104 @@
...
 
 #### Frontier chains
 To contribute a frontier OP-Stack chain configuration, you set the `SCR_CHAIN_TYPE=frontier` in the `.env` file. This will set `superchain_level = 1` in the registry's config file for this chain.
 
 #### Standard chains
+
+todo: is this different from frontier chains? and which tool are we referring to?
```
Comment (tessr) [inline README.md:76]: Can we get this todo resolved? What flags do we need to set in the env file?

File: README.md [hunk 3]
```diff
@@ -6,79 +6,104 @@
...
+To contribute a standard OP-Stack chain configuration, in addition to user-supplied metadata (chain name) the following data is required: contracts deployment, rollup config, L2 genesis. We provide a tool to scrape this information from your local [monorepo](https://github.com/ethereum-optimism/optimism) folder.
+
 Adding a standard chain is a two-step process. First, the chain should be added as a frontier chain as above, but with `SCR_STANDARD_CHAIN_CANDIDATE=true` in the `.env` file.
```
Comment (tessr) [inline README.md:79]: This is actually kind of weird, that you have to set `SCR_CHAIN_TYPE=frontier` for standard chains. Presumably these get turned into numerical levels on the other end, but it's weird that we ask people to type out `frontier` for their standard candidate chains

File: README.md [hunk 4]
```diff
@@ -6,79 +6,104 @@
...
-The Superchain Registry repository hosts Superchain-configuration data in a minimal human-readable form.
-
-This includes mainnet and testnet Superchain targets, and their respective member chains.
+The Superchain Registry is an index of chains which serves as the source of truth for who’s in the Superchain and what modifications they’ve made to their chains. 
+It hosts Superchain-configuration data in a minimal human-readable form and includes mainnet and testnet Superchain targets, along with their respective member chains.
 
 Other configuration, such as contract-permissions and `SystemConfig` parameters are hosted and governed onchain.
 
-A list of chains in the registry can be seen in the top level [`chainList.toml`](./chainList.toml) and [`chainList.json`](./chainList.json) files.
+A list of chains in the registry can be seen in the top level [`chainList.toml`](./chainList.toml) and [`chainList.json`](./chainList.json) files.
```
Comment (bitwiseguy) [inline README.md:14]: Do we intend to keep both of these files going forward? Seems like it would be better to pick one and deduplicate the data

File: README.md [hunk 5]
```diff
@@ -6,79 +6,104 @@
...
+To help with clarity and a common understanding, here are some helpful terms and their definitions.
+
+### General
+
+**Superchain member:** A chain with an agreement in place to commit sequencer revenue back to the Optimism Collective.  
+**Blockspace charter:** A [Blockspace Charter](https://gov.optimism.io/t/season-6-introducing-blockspace-charters-superchain-first-governance/8133) is a technical-focused governing document (and framework) for the Superchain.
+**Standard chain:** A chain that conform to the [Standard Rollup Charter - not ratified yet](https://gov.optimism.io/t/season-6-draft-standard-rollup-charter/8135)
+**Frontier chain:** A non-standard chain which has modifications that do not fit the `Standard Rollup Charter` criteria. 
+**Standard chain candidate:** A chain that has met most of the standard chain criteria, except for the `ProxyAdminOwner` key handover.    
+**Key handover:**  A colloquial term for updating the chain's `ProxyAdminOwner` to fufil the requirements from the standard rollup charter.
```
Comment (bitwiseguy) [inline README.md:28]: Typo: fulfil --> fulfill

File: README.md [hunk 6]
```diff
@@ -6,79 +6,104 @@
...
-> **Note**
-> Hardfork override times, where they have been set, will be included. If and when a chain becomes a standard chain, a `superchain_time` is set in the chain config. From that time on, future hardfork activation times which are missing from the chain config will be inherited from superchain-wide values in the neighboring `superchain.yaml` file.
-
-- Addresses of L1 contracts. (Note that all L2 addresses are statically known addresses defined in the OP-Stack specification, and thus not configured per chain.) These are written to `extra/addresses/superchain_target/chain_short_name.json`.
+- Hardfork override times, where they have been set, will be included. If and when a chain becomes a standard chain, a `superchain_time` is set in the chain config. From that time on, future hardfork activation times which are missing from the chain config will be inherited from superchain-wide values in the neighboring `superchain.yaml` file.
+- Addresses of L1 contracts are written to `extra/addresses/superchain_target/chain_short_name.json`. Note that all L2 addresses are statically known addresses defined in the OP-Stack specification, and thus not configured per chain.
```
Comment (bitwiseguy) [inline README.md:95]: This is no longer true after the repo file structure refactor. Addresses will not be written to `superchain/config/<chain-short-name>.toml` (in the field titled `[addresses]`) (e.g. [here](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/mainnet/op.toml#L31))

File: README.md [hunk 7]
```diff
@@ -6,79 +6,104 @@
...
+### 4. Understand output
 The tool will write the following data:
 - The main configuration source, with genesis data, and address of onchain system configuration. These are written to `superchain/configs/superchain_target/chain_short_name.yaml`.
-> **Note**
-> Hardfork override times, where they have been set, will be included. If and when a chain becomes a standard chain, a `superchain_time` is set in the chain config. From that time on, future hardfork activation times which are missing from the chain config will be inherited from superchain-wide values in the neighboring `superchain.yaml` file.
-
-- Addresses of L1 contracts. (Note that all L2 addresses are statically known addresses defined in the OP-Stack specification, and thus not configured per chain.) These are written to `extra/addresses/superchain_target/chain_short_name.json`.
+- Hardfork override times, where they have been set, will be included. If and when a chain becomes a standard chain, a `superchain_time` is set in the chain config. From that time on, future hardfork activation times which are missing from the chain config will be inherited from superchain-wide values in the neighboring `superchain.yaml` file.
```
Comment (bitwiseguy) [inline README.md:94]: The neighboring file is now `superchain.toml` instead of `superchain.yaml`

File: README.md [hunk 8]
```diff
@@ -6,79 +6,104 @@
...
 ```shell
 just add-chain
 ```
 
 The remaining steps should then be followed to merge the config data into the registry -- a prerequisite for [promoting the chain](#promote-a-chain-to-standard) to a standard chain.
 
-### 5. Understand output
+### 4. Understand output
 The tool will write the following data:
 - The main configuration source, with genesis data, and address of onchain system configuration. These are written to `superchain/configs/superchain_target/chain_short_name.yaml`.
```
Comment (bitwiseguy) [inline README.md:93]: `superchain/configs/superchain_target/chain_short_name.yaml` --> `superchain/configs/<superchain-target>/<chain-short-name>.toml`

File: README.md [hunk 9]
```diff
@@ -92,22 +117,25 @@ just validate <chain-id>
 
 The [`validation_test.go`](./validation/validation_test.go) test declaration file defines which checks run on each class of chain. The parameters referenced in each check are recorded in [TOML files in the `standard` directory](./validation/standard).
 
-### 7. Run codegen and check output
-This is a tool which will rewrite certain summary files of all the chains in the registry, including the one you are adding. The output will be checked in a continuous integration checks (it is required to pass):
+### 6. Run codegen and check output
+This is a tool which will rewrite certain summary files of all the chains in the registry, including the one you are adding.
```
Comment (bitwiseguy) [inline README.md:121]: Suggest rewording this to make it a bit more direct:
```
This tool will add your chain to the chainList.toml and addresses.json files, which contain summaries of all chains in the registry.
```

File: README.md [hunk 10]
```diff
@@ -92,22 +117,25 @@ just validate <chain-id>
...
+> If anything looks incorrect, please get in touch.
 
-### 8. Open Your Pull Request
+### 7. Open Your Pull Request
 When opening a PR:
 - Open it from a non-protected branch in your fork (e.g. avoid the `main` branch). This allows maintainers to push to your branch if needed, which streamlines the review and merge process.
 - Open one PR per chain you would like to add. This ensures the merge of one chain is not blocked by unexpected issues.
 
-Once the PR is opened, the same automated checks from Step 4 will then run on your PR, and your PR will be reviewed in due course. Once these checks pass the PR will be merged.
+Once the PR is opened, the same automated checks from Step 5 will then run on your PR, and your PR will be reviewed in due course. Once these checks pass the PR will be merged.
```
Comment (bitwiseguy) [inline README.md:138]: Suggest adding comma: `Once these checks pass, the PR will be merged`

File: superchain/configs/README.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+- genesis:
+  - l1
+    - hash
+    - number
+  - l2:
+    - hash
+    - number
+  - l2_time
+- block_time
+- seq_window_size
```
Comment (bitwiseguy) [inline superchain/configs/README.md:28]: This format has changed, and will likely continue to evolve in the future as we add fields. Maybe we could link to one example like `mainnet/op.toml` so we don't have to update this README whenever we make changes?

File: superchain/extra/README.MD [hunk 12]
```diff
@@ -1,4 +1,8 @@
 # extra
 
-This is a collection of extra configuration,
-since not all address and system-config information is available through L1 for some chains.
+This is a collection of extra configuration data because not all address and system-config information is available through L1 for some chains.
+
+- [addresses](./addresses/): L1 Smart Contract addresses on for each network, grouped by Superchain target.
```
Comment (bitwiseguy) [inline superchain/extra/README.MD:5]: This information now lives in the toml config files so this dir should not be referenced

File: superchain/extra/README.MD [hunk 13]
```diff
@@ -1,4 +1,8 @@
...
+- [bytecodes](./bytecodes/): The bytecode that is pulled into the compressed genesis system configuration file.
+- [genesis](./genesis/): Compressed genesis system configuration data.
+- [genesis-system-configs](./genesis-system-configs/): Networks additional system configuration information.
```
Comment (bitwiseguy) [inline superchain/extra/README.MD:8]: This information now lives in the toml config files so this dir should not be referenced

File: README.md (hunk 4 above)
Comment (geoknee) [reply README.md:14]: I think we decided to use both for now.

File: superchain/extra/README.MD [hunk 14]
```diff
@@ -1,4 +1,7 @@
 # extra
 
-This is a collection of extra configuration,
-since not all address and system-config information is available through L1 for some chains.
+This is a collection of extra configuration data because not all address and system-config information is available through L1 for some chains.
+
+- [addresses.json](./addresses/addresses.json): L1 Smart Contract addresses for each network.
+- [bytecodes](./bytecodes/): The bytecode that is pulled into the compressed genesis system configuration file.
```
Comment (geoknee) [inline superchain/extra/README.MD:6]:
```suggestion
- [bytecodes](./bytecodes/): The full bytecode that is factored out of compressed genesis system configuration file. Not designed for human consumption.
```

File: superchain/extra/README.MD [hunk 15]
```diff
@@ -1,4 +1,7 @@
...
+- [genesis](./genesis/): Compressed genesis system configuration data.
```
Comment (geoknee) [inline superchain/extra/README.MD:7]:
```suggestion
- [genesis](./genesis/): Compressed genesis system configuration data. Not designed for human consumption.
```

File: validation/README.md [hunk 16]
```diff
@@ -0,0 +1,4 @@
+# validation
+
+This directory contains the golang validation checks that chains run against.
+The [validation_test.go](validation/validation_test.go) file is the entry point of the tests. It determines to run the universal and additions tests for Standard Candidates and Standard chains.
```
Comment (geoknee) [inline validation/README.md:4]:
```suggestion
The [validation_test.go](validation/validation_test.go) file is the entry point of the tests. It runs a different set of tests for each class of chain (Frontier, Standard Candidate and Standard).
```

**Review by geoknee — APPROVED** [review]
